src/main.ts

```diff
@@ -1,5 +1,7 @@
 import * as globby from 'globby';
 
+let STORE: any = {};
```

Comment:
Prefer `const` over `let`. We can initialize the `STORE` variable here.

```ts
const STORE = {
  context: {},
  reporters: [],
};
```

We can use an array to hold multiple reporters. Reporters will receive events when tests are executed (test-start, test-complete, test-error, action-start, action-error, etc.)

Reply:
Done, thanks